src/modules/current.js

```diff
@@ -37,7 +43,21 @@ export const getProgramStageFromUi = (ui) =>
           }
         : {}
 
-export const getOptionsFromUi = (ui) => pick(ui.options, Object.keys(options))
+export const getOptionsFromUi = (ui) => {
+    const savableOptions = pick(
+        ui.options,
+        Object.keys(options).filter((option) => options[option].savable)
```

Comment:
This is a "breaking change" for this function, as it does not return the same set of options.
I'm not sure if we can change it like this.
It might be fine for LL, but the idea (at least in DV) I think is that all options go into `current` which is passed to the plugin and when saving, the `savable` options are the only ones that end up in the object POSTed to the endpoint.

Reply:
Same thing below.